imjoy/imjoyPluginEngine.py

```diff
@@ -139,12 +139,19 @@
     logger.setLevel(logging.ERROR)
 
 
-if opt.serve and os.path.exists(os.path.join(WEB_APP_DIR, 'index.html')) and os.path.exists(os.path.join(WEB_APP_DIR, 'static')):
+if opt.serve and os.path.exists(os.path.join(WEB_APP_DIR, 'index.html')) \
+        and os.path.exists(os.path.join(WEB_APP_DIR, 'static')) \
+        and os.path.exists(os.path.join(WEB_APP_DIR, 'css')) \
+        and os.path.exists(os.path.join(WEB_APP_DIR, 'js')) \
+        and os.path.exists(os.path.join(WEB_APP_DIR, 'docs')):
     async def index(request):
         """Serve the client-side application."""
         with open(os.path.join(WEB_APP_DIR, 'index.html'), 'r', encoding="utf-8") as f:
             return web.Response(text=f.read(), content_type='text/html')
     app.router.add_static('/static', path=str(os.path.join(WEB_APP_DIR, 'static')))
+    app.router.add_static('/css', path=str(os.path.join(WEB_APP_DIR, 'css')))
+    app.router.add_static('/js', path=str(os.path.join(WEB_APP_DIR, 'js')))
```

Comment:
With this https://github.com/oeway/ImJoy/pull/41 , `/css` and `/js` would not be necessary, they will be moved to `/static`.

But we can keep `/docs`.